wordle-web/src/components/KeyBoard.vue

```diff
@@ -1,48 +1,90 @@
 <template>
   <v-row class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
-    <v-col v-for="(letter, j) in key" :key="j">
-      <LetterButton :letter="letter" @click="letterClick(letter)" />
+    <v-col cols="auto" v-for="(letter, j) in key" :key="j">
```

Comment:
Your keyboard has a bit of a weird styling
![image](https://user-images.githubusercontent.com/43832670/236076902-4c321c1d-2cf0-4918-846d-a012b2be1f41.png)